docs/samples/Microsoft.ML.Samples/Dynamic/Calibrator.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.ML.Calibrator;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class CalibratorExample
+    {
```

Comment:
It'd be better to explain the whole story about this example here. Calibrator in ML.NET is used to produce probability output (e.g., in binary classification, it can be the probability of finding a positive label). This example first trains a support vector machine and then convert its output to probability via training a calibrator.  The overall pipeline looks like
input text ---> feature engineering (FeaturizeText) ---> feature vector ---> SVM (StochasticDualCoordinateAscent) ---> score ---> Calibrator (PlattCalibratorEstimator) ---> probability.

Reply:
I am torn on putting this much text here, because this example will live in a page with documentation about the calibrators. 

Leaving it open, to see what is the average amt of explanation in the docs. 

---
In reply to: [240318918](https://github.com/dotnet/machinelearning/pull/1816#discussion_r240318918) [](ancestors = 240318918)

Reply:
I might move the explanation to base class, that reference this example. 

---
In reply to: [240426286](https://github.com/dotnet/machinelearning/pull/1816#discussion_r240426286) [](ancestors = 240426286,240318918)

Reply:
Please at least explain the pipeline structure trained in this example. An example needs to be self-contained.

---
In reply to: [240428293](https://github.com/dotnet/machinelearning/pull/1816#discussion_r240428293) [](ancestors = 240428293,240426286,240318918)

Reply:
I broke down the comments putting them some in the header of this file, some in the CalibratorsCatalog, and some in the body of the example, so they are closer to the action they describe. 

---
In reply to: [240428293](https://github.com/dotnet/machinelearning/pull/1816#discussion_r240428293) [](ancestors = 240428293,240426286,240318918)